lib/components/atoms/Blockquote/tests/__snapshots__/Blockquote.test.js.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Blockquote /> should render correctly 1`] = `
+<StyledComponent
```

Comment:
You are generating snapshot of wrong component, snapshot should display you markup and not object